.github/workflows/codeql.yml

```diff
@@ -1,96 +1,46 @@
-# For most projects, this workflow file will not need changing; you simply need
-# to commit it to your repository.
-#
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
-#
-# ******** NOTE ********
-# We have attempted to detect the languages in your repository. Please check
-# the `language` matrix defined below to confirm you have the correct set of
-# supported CodeQL languages.
-#
-name: "CodeQL"
+name: "Code Scanning - Action"
 
 on:
   push:
-    branches: [ "main", "extensions/rc1", "master", "release/*", "release/extensions" ]
-  pull_request:
-    branches: [ "main", "extensions/rc1", "master", "release/*", "release/extensions" ]
   schedule:
-    - cron: '35 3 * * 1'
-  workflow_dispatch:
+    - cron: '0 0 * * 0'
 
 jobs:
-  analyze:
-    name: Analyze (${{ matrix.language }})
-    # Runner size impacts CodeQL analysis time. To learn more, please see:
```

Comment:
Why this change on codeql file? Is it required ?

Reply:
Not required, reverted.